zcash_primitives/src/transaction/builder.rs

```diff
@@ -522,20 +528,22 @@ impl<'a, P: consensus::Parameters, U: sapling::builder::ProverProgress> Builder<
                 transparent_inputs,
                 self.transparent_builder.outputs(),
                 sapling_spends,
```

Comment:
Non-blocking: I think we need to alter this to call `bundle_type.num_spends()` after the changes in https://github.com/zcash/sapling-crypto/pull/114. This should happen in a separate PR.

Reply:
That is happening as part of #1060